Create .gitbook.yaml (#313)
mattermost/mattermost-plugin-incident-response: docs/incidents/ending-an-incident.md

# Ending an Incident

> The owner of the incident can end it via either a slash command or the right-hand sidebar.

Reviewer: Propose adding something akin to:

> Ending an incident signals to all participants that the issue has been resolved. Some playbooks may define stages and tasks to complete after an incident has been resolved, such as scheduling and completing a post-mortem. The incident RHS remains fully accessible even after an incident has ended.

Author: That's perfect, thank you!

# Ending an Incident

> The owner of the incident can end it via either a slash command or the right-hand sidebar.

Reviewer 2: I'm not exactly sure if you mean commander with "incident owner", but every incident member can end the incident.

> The owner of the incident can end it via either a slash command or the right-hand sidebar.

Author: ```suggestion
Incident members can end an incident via either a slash command or the right-hand sidebar.
```
